oak_enclave_binary/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "oak_enclave_shim"
+name = "oak_enclave_binary"
```

Comment:
I think just calling it `oak_enclave` is shorter and prettier.
But that's a bikeshedding for another time =)